hack/ci-test-e2e.sh

```diff
@@ -31,6 +31,8 @@ bin/test-e2e \
   -test.parallel=20 \
   --e2e.aws-credentials-file=/etc/hypershift-pool-aws-credentials/credentials \
```

Comment:
Similar comment as above:

Maybe change the hardcoded variable values to be like an environment variable as well to keep it consistent with the other variables?

Reply:
This is certainly possible if I shift these variables to https://github.com/openshift/release/blob/6ecb6a493dd71ade1e42de62f893bd5395cf7859/ci-operator/step-registry/hypershift/aws/run-e2e/hypershift-aws-run-e2e-chain.yaml#L14-L20, do we want to do that? I think the majority are static here with the exception of `OCP_IMAGE_LATEST`

Reply:
Oh nvm, I misread the file extension and was thinking it was a .md. Ignore my comment here.